packages/report-server/src/routes/FetchTransformSchemaRoute.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+
+import { Route } from '@tupaia/server-boilerplate';
+
+import { TransformSchema } from '../types';
+import { transformSchemas } from '../reportBuilder/transform/functions';
+import { aliases } from '../reportBuilder/transform/aliases';
+
+export type FetchTransformSchemaRequest = Request<
+  Record<string, never>,
+  TransformSchema[],
+  Record<string, never>,
+  Record<string, never>
+>;
+
+export class FetchTransformSchemaRoute extends Route<FetchTransformSchemaRequest> {
+  async buildResponse() {
+    const formattedTransformSchema = Object.entries(transformSchemas).map(
+      ([transformKey, config]) => {
+        const { fields } = config;
+        return {
+          name: transformKey,
+          code: transformKey,
+          ...fields,
+          where: {
+            type: 'string',
+            optional: true,
+          },
```

Comment:
I don't think this is quite accurate. The `where` clause doesn't apply to all transforms (I think `sortRows` doesn't use it). Moreso, if it does apply, it ought to be in the `paramsValidator` for that transform right? So we can omit this?